addons/project/controllers/portal.py

```diff
@@ -144,10 +144,11 @@ def _prepare_project_sharing_session_info(self, project, task=None):
             "translations": translation_hash,
         }
 
+        action_mod = project.action_project_sharing()
         project_company = project.company_id
         session_info.update(
             cache_hashes=cache_hashes,
-            action_name='project.project_sharing_project_task_action',
+            action_name=action_mod,
```

Comment:
```suggestion
            action_name=project.action_project_sharing(),
```
No need for a variable